includes/rest-api.php

```diff
@@ -149,15 +151,17 @@ function register_rest_routes() {
 		'distributor/list-pull-content',
 		array(
 			'methods'             => 'POST',
-			'callback'            => __NAMESPACE__ . '\get_pull_content',
+			'callback'            => __NAMESPACE__ . '\get_pull_content_list',
```

Comment:
@peterwilsoncc Can we use a double slash as we are doing in the following line?

Reply:
62d165fe17ce35010d197ebdd4e076800f96a311